Ground-Items.md

```diff
@@ -118,20 +121,47 @@ Configures the color for insane value items.
 
 Configures the start price for insane value items.
 
-### Only show loot
+### Ownership filter
 
-Only shows drops from NPCs and players.
+Options for deciding which type of loot displays.
 
-### Delay for double-tap ALT to hide
+All - All
+Takeable - Any Loot/Items you can pick up, including your (GIM)group loot
+Drops - Enemy drops for you, and your (GIM)group loot
 
-Decrease this number if you accidentally hide ground items often. `0` = Disabled.
 
-### Collapse ground item menu entries
+### Double-tap delay
+
+Delay for double-tapping `ALT` to hide. Decrease this number if you accidentally hide ground items often. `0` = Disabled.
+
+### Collapse ground item menu
 
 Collapses ground item menu entries together and appends count.
 
-### Show despawn timers
+### Despawn timer
 
 Shows a green timer ![Green Timer](img/ground-items/ground_items_green_timer.png) that counts down until items dropped or received as loot are visible to other players.
 
-Shows a yellow timer ![Yellow Timer](img/ground-items/ground_items_yellow_timer.png) that counts down until items dropped or received as loot despawn.
+Shows a yellow timer ![Yellow Timer](img/ground-items/ground_items_yellow_timer.png) that counts down until items dropped or received as loot, despawn.
```

Comment:
Idk if this is pie only, or also happens with seconds/ticks enabled. Probably does both? Might be worth mentioning ticks/seconds in that case. If it does not do the pie with ticks/seconds selected, it'd require some changes anyway.

Reply:
added only seconds cuz frick ticks